packages/eas-cli/src/build/create.ts

```diff
@@ -104,7 +115,11 @@ async function waitForBuildEndAsync(
           break;
         case BuildStatus.ERRORED:
           spinner.fail('Build failed');
-          throw new Error(`Standalone build failed!`);
+          if (build.error) {
```

Comment:
What about the case when building with `-p all` ?

Reply:
If you look closer at the code you notice that this `switch` is only used for building single-platform builds. I.e. it's not used for the `-p all` case.

Reply:
yes that is my point, we are not displaying those errors when building for multiple platforms

Reply:
Actually, we do display errors when building for multiple platforms. See also the screenshots in the PR description :P